### DISCUSSION TITLE
Problemi installazione e configurazione su SSD di Raspbian e HA CORE

### DISCUSSION THREAD

**Discussion by braudoo** [indomus/forum#1129] [HUB: Home Assistant]
Salve a tutti,
seguendo la guida 
https://indomus.it/guide/installare-e-avviare-home-assistant-os-su-ssd-esterno-su-raspberry-pi-4/
volevo sapere se era possibile installare HA core in ambiente venv anche su disco SSD, in quanto con questa guida si installerebbe solamente HASSIO.

Ho provato a seguire la seguente guida
https://indomus.it/progetti/avviare-raspberry-pi-tramite-disco-esterno-ssd-o-altro/
e poi tramite SSH seguire la procedura per l'installazione di HA Core, ma all'atto di installare Python mi generava una serie di errori inerenti a `libturbojpeg0-dev`.

Qualcuno riuscirebbe ad aiutarmi?

Grazie.

**Comment by snakuzzo** [comment]
Certo che puoi installare Core su venv su un SSD, ma se non specifichi bene tutti i passaggi che fai e gli errori che hai, nessuno può darti una mano. Non basta dire "Python mi generava una serie di errori..."

**Comment by braudoo** [comment]
Ciao grazie per la risposta.
Qui di seguito la sequenza del comando di installazione di Python e dell'errore che mi da:

```
sudo apt-get install python3 python3-dev python3-venv python3-pip libffi-dev libssl-dev libjpeg-dev zlib1g-dev autoconf build-essential libopenjp2-7 libtiff5 libturbojpeg tzdata
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Package libturbojpeg is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  libturbojpeg0
E: Package 'libturbojpeg' has no installation candidate
```

Ho tentato anche di installare 'libturbojpeg' ma questo è quanto mi restituisce

```
pi@raspberrypi:~ $ sudo apt-get install -y libturbojpeg
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Package libturbojpeg is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  libturbojpeg0

E: Package 'libturbojpeg' has no installation candidate
```
Purtroppo mi trovo in un vicolo ceco.

Al momento sto utilizzando HASSIO.

**Reply by snakuzzo** [reply]
Scusa ma che stai combinando ?
Che c'entra l'installazione di pacchetti se sei su Hassio ?
Sicuro di avere Hassio?  A me non sembra :)

ad ogni modo...se sei su raspberry os... che sistema operativo stai usando?
Lancia

`lsb_release -a`

e comunque prova ad installare questo...

`sudo apt-get install -y libturbojpeg-dev`

**Reply by braudoo** [reply]
Scusami ti ho fatto confusione io...
Su un disco SSD ho installato Raspberry OS dove ho fatto i passaggi elencati sopra.
Su un altro disco SSD ho installato HASSIO seguendo la guida. 
Per questo ti ho scritto così.
Il mio obiettivo è installare HA CORE dove attualmente ho installato Raspbian.

Faccio il cambio di SSD e poi provo. Grazie

**Comment by braudoo** [comment]
Eccomi, 
allora questa è la mia versione

```
pi@raspberrypi:~ $ lsb_release -a
No LSB modules are available.
Distributor ID: Raspbian
Description:    Raspbian GNU/Linux 11 (bullseye)
Release:        11
Codename:       bullseye
pi@raspberrypi:~ $
```
Lanciando il comando `sudo apt-get install -y libturbojpeg-dev`
mi restituisce questo

```
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'libturbojpeg0-dev' instead of 'libturbojpeg-dev'
libturbojpeg0-dev is already the newest version (1:2.0.6-4).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.

```

Ma come solito lanciando il comando di installazione Python mi restituisce sempre

```
Package libturbojpeg is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  libturbojpeg0

E: Package 'libturbojpeg' has no installation candidate
```

Hai qualche suggerimento?

Grazie

**Reply by snakuzzo** [reply]
Mi dici quale comando lanci che ti restituisce quell'errore?

**Reply by braudoo** [reply]
Il comando è quello nella guida 

https://indomus.it/guide/come-installare-e-configurare-home-assistant-su-un-raspberry-pi-gia-in-uso/

E lanciando il comando:
`sudo apt-get install python3 python3-dev python3-venv python3-pip libffi-dev libssl-dev libjpeg-dev zlib1g-dev autoconf build-essential libopenjp2-7 libtiff5 libturbojpeg tzdata`

Mi restituisce l'errore.

Premetto che ho fatto tutti gli aggiornamenti correttamente.

**Reply by Scialla** [reply]
a me sembra ti dica cosa devi fare, installare `libturbojpeg0` al posto di `libturbojpeg`

**Reply by snakuzzo** [reply]
Sì, mi sa che hai ragione...non avevo letto con attenzione

**Comment by braudoo** [comment]
Ciao sono riuscito a completare installazione dopo aver installato libturbojpeg0.

Grazie
